db/migrations/0003_alter_skill_race.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-10-18 10:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_guild_description'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
```

Comment:
The use of `on_delete=django.db.models.deletion.CASCADE` should be carefully considered. According to the checklist, you should use `on_delete=models.CASCADE` if the 'skill' model cannot exist without the 'race' model. Ensure this is the intended behavior, as this will delete all 'skill' instances related to a 'race' when that 'race' is deleted.